webpack.config.js

```diff
@@ -109,23 +109,29 @@ module.exports = (env = {}) => {
     module: {
       rules: [
         {
-          test: /\.js$/,
+          test: /\.(ts|js)x?$/,
           exclude: /node_modules/,
           use: [
             {
               loader: 'babel-loader',
               options: {
                 presets: [
                   [
-                    'env',
+                    '@babel/preset-env',
```

Comment:
Since you made a `babel.config.js` file, can you move all these options to that file?

Maybe the `isTest` part needs to stay here. Or maybe there's a way to load that from env vars outside the webpack config.